crates/walrus-service/src/backup/garbage_collector.rs

```diff
@@ -65,6 +65,31 @@ async fn collect_garbage(
         .await
         .expect("failed to connect to postgres for collect_garbage");
 
+    let garbage_query = format!(
+        "
+            WITH expired_blob_ids AS (
+                SELECT blob_id
+                FROM blob_state
+                WHERE
+                    state = 'archived' AND
+                    (initiate_gc_after IS NULL OR initiate_gc_after < NOW()) AND
+                    COALESCE(
+                        end_epoch + {} <= (
```

Comment:
Is the `+ {}` the only part changed in the query, is that right?

Reply:
Yes, that's correct.